lib/core/test/unit/Cardano/Wallet/DB/Properties.hs

```diff
@@ -1068,3 +1069,7 @@ genBatches (GenSparseCheckpointsArgs cfg h) = do
         -- sure we generate realistic cases.
         n <- fromIntegral <$> choose (1, 3 * gapSize cfg)
         go (drop n source) (take n source : batches)
+
+-- | Microseconds in a minute
+minute :: Int
+minute = 60*1000*1000
```

Comment:
Suggestion: use `NumericUnderscores` (and use a plural "minutes").

```suggestion
minutes :: Int
minutes = 60_000_000
```

Reply:
NumericUnderscores — sure. Not sure we should use plural though? 

I know we have decided to use singular for units in the context of `Quantity` at least:

```shell
$ git grep "Quantity \"slot"                                                                                                                                                                                                                               
lib/core/src/Cardano/Wallet/Api/Types.hs:    , epochLength :: !(Quantity "slot" Word32)
lib/core/src/Cardano/Wallet/Primitive/Slotting.hs:slotDifference :: SlotParameters -> SlotId -> SlotId -> Quantity "slot" Natural
lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs:instance Arbitrary (Quantity "slot" Word32) where
lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs:instance Arbitrary (Quantity "slot" Natural) where
```